amgut/handlers/auth_handlers.py

```diff
@@ -66,12 +79,23 @@ def post(self):
             logging.exception('Error on skid %s:' % skid)
 
 
-class AuthLoginHandler(BaseHandler):
+class AuthLoginHandler(AuthBasehandler):
     """user login, no page necessary"""
+    def get(self, *args, **kwargs):
```

Comment:
They get passed in by tornado if they're in the URL, so you get a `TypeError` due to an incorrect number of args